tests/ExternalActiveRelationTest.php

```diff
@@ -49,14 +57,14 @@ public function testFindEager()
     {
         // has one :
         $articles = ArticleIndex::find()->with('source')->all();
-        $this->assertEquals(2, count($articles));
+        $this->assertEquals($this->pagination->defaultPageSize, count($articles));
```

Comment:
This is not a proper assertion.
Exact number match should be used as it was before.